config/forms/2021_2022.json

```diff
@@ -1147,7 +1147,12 @@
                       0
                     ]
                   },
-                  "hidden_in_check_answers": true
+                  "hidden_in_check_answers": {
+                    "depends_on": [
+                      { "age1_known": 0 },
+                      { "age1_known": 1 }
+                    ]
+                  }
```

Comment:
How does this change from the behaviour we have now?

Reply:
Right now it means that this question will only be hidden if the `age1_known` has been answered. This wasn't the case before. You could navigate away and the age question would be hidden on check answers

Reply:
What's the reason for not hiding it all the time?

Reply:
If it is hidden all the time you get a bug where you can't answer the lead tenant age question if you only answer the number of household members question and then navigate away from the page

Reply:
But wouldn't you see the "tenant age" question? Happy with the change if not 🤷‍♀️

Reply:
Before:
<img width="871" alt="Screenshot 2022-04-14 at 15 03 34" src="https://user-images.githubusercontent.com/47317567/163407482-dc46344c-18ed-4143-b655-2cc8f2521779.png">

After:
<img width="860" alt="Screenshot 2022-04-14 at 15 02 03" src="https://user-images.githubusercontent.com/47317567/163407521-9a9ac9e2-5ea8-4c5d-ad43-02bb2c75896d.png">

Reply:
Allowing you to go back and answer no for example which then gives you this
<img width="855" alt="Screenshot 2022-04-14 at 15 05 27" src="https://user-images.githubusercontent.com/47317567/163407815-9e4f36de-0d40-4664-9478-51394deda826.png">
: